yarn.lock

```diff
@@ -6182,7 +6182,7 @@
   dependencies:
     "@types/node" "*"
 
-"@types/http-proxy-middleware@*", "@types/http-proxy-middleware@^0.19.3":
+"@types/http-proxy-middleware@^0.19.3":
```

Comment:
Is this still needed? Looks like `http-proxy-middleware` now brings it's own typings.

Reply:
Ah. So maybe it's ripe for removal then